### ISSUE TITLE
Cannot get `RELOADIUMPATH` to be picked up by PyCharm

### ISSUE THREAD

**Issue by wrobelda** [reloadware/reloadium#50]
**Describe the bug**
I am trying to set up `RELOADIUMPATH` env variable in Pycharm, but consistently am getting the "No files are watched."

Tried:
- adding it in Run/Debug Configuration
- adding it in Preferences -> Build, Execution, Deployment -> Console -> Python Console and editing Run/Debug Configuration to Run with Python Console 
  - confirmed with `os.environ` in Python Console window that the variable is set up as expected

I would suggest that you improve the error message's verbosity to actually  inform whether `RELOADIUMPATH` was set and if so, to which value. Right now it's hard to tell where to look for the cause. 

**Expected behavior**
`RELOADIUMPATH` set in Run/Debug Configuration should be picked up.
Error message should be more verbose. 

**Desktop (please complete the following information):**
 - OS: macOS
 - OS version: 12.6
 - Reloadium package version: 0.9.3
 - PyCharm plugin version: 0.8.6
 - Editor: PyCharm 
 - Run mode: Run/Debug

Actual console log output:
```
Loaded 0 watched modules so far from paths:
 - /Users/cromo/Documents/Sourcecode/woob/venv/bin/**/*.py

No files are watched.
Editing files will have no effect!
Only files inside current working directory or directories specified in RELOADIUMPATH environmental variable are watched.
```

**Comment by dkrystki** [comment]
You should add your paths to "Reloadable Paths" if you're using the plugin.
I'll make it more clear in the documentation.
![2022-10-12_20:12:01](https://user-images.githubusercontent.com/45499236/195316403-4cf97a22-9137-4eb8-a7de-1ccdd0ef71b2.png)

**Comment by wrobelda** [comment]
Oh wow, I missed that completely. However, since "Add sources roots to reloadable paths" is checked, my understanding is that all of the project's sources should be included by default, too? Cause they seem to be ignored here. 
Explicitly adding a path to the module I am working on though works just fine.

Appreciate quick response!

**Comment by dkrystki** [comment]
For "Add sources roots to reloadable paths" you will have to do right click -> "Mark directory as" -> "Sources root"
Then it'll turn blue and be reloadable.
![2022-10-12_20:31:17](https://user-images.githubusercontent.com/45499236/195320447-e092f688-7887-4050-8784-f97617bd45b0.png)

**Comment by wrobelda** [comment]
Ah, OK, there goes my somewhat limited knowledge of PyCharm. Thanks again for you help!

**Comment by dkrystki** [comment]
Fixed in reloadium 0.9.4 and PyCharm plugin 0.8.8
